workspace/src/app/views/taskViews/transform/evalution/TransformRuleNodeEvaluation.tsx

```diff
@@ -0,0 +1,88 @@
+import { WhiteSpaceContainer, Tag, Tooltip } from "@eccenca/gui-elements";
```

Comment:
Validation errors are currently not shown for transform rules. Can you change this?
Can you also delete `TransformRuleNodeEvaluation` and instead adapt `LinkRuleNodeEvaluation.tsx` to be used for transform and linking rules?